src/yashCJun20/Assignment22/VerifyPalindromeNumber.java

```diff
@@ -0,0 +1,25 @@
+package yashCJun20.Assignment22;
+class VerifyPalindromeNumber{
+	void toVerifyPalindromeNumberOrNot(int num){
+		int remender;
+		int reverseNumber=0;
+		int originalNumber;
```

Comment:
Directly initialize the value to the local variable. 
int originalNumber = num;